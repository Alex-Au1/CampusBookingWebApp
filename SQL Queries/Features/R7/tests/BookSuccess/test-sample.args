{
    "args": [
        "Jane Doe",
        "RCH 105",
        "2028-01-17 10:30:05",
        "2028-01-17 11:30:19",
        3
    ],
    "kwargs": {}
}