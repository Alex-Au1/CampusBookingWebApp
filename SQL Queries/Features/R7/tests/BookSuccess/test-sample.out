Invalid UUID format for user ID or room ID.